sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -992,6 +992,9 @@ case class FormatNumber(x: Expression, d: Expression)
 
       val sb = classOf[StringBuffer].getName
       val df = classOf[DecimalFormat].getName
+      val dfs = classOf[DecimalFormatSymbols].getName
+      val l = classOf[Locale].getName
+      val US = "US"
```

Comment:
It's not a big deal but my tastes would run opposite. This is another layer of indirection and the variable "US" to represent the string "US" doesn't seem to add anything. At least I might call it "locale" or something. "l" might be better as "localeClass"